roles/nova-common/defaults/main.yml

```diff
@@ -20,6 +20,7 @@ nova:
       config_file: libvirt.conf
       defaults: /etc/sysconfig/libvirtd
       group: libvirt
+  nova_lib_dir: "/opt/openstack/current/nova"
```

Comment:
Also we need to ensure following for distro path
- ensure openstack-nova-placement rpm is being installed  (will have to check for exact rpm name). This is done by adding rpm to project_packages list in nova-common/defaults (for distro)
- add nova placement service defn to https://github.com/blueboxgroup/ursula/blob/master/roles/openstack-meta/defaults/main.yml#L113  . Follow existing pattern to stay consistent.
- ensure systemd override is getting written out - https://github.com/blueboxgroup/ursula/blob/master/roles/nova-control/tasks/main.yml#L23 - Follow same pattern, we write override systemd defn in /etc/systemd.